src/elements/content-uploader/UploadInput.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable */ // Disabling linting for tabIndex on label, unknown why it's there
+import * as React from 'react';
```

Comment:
are we able to disable the specific error instead of the whole file?

Reply:
this was the alt 
`// eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex,jsx-a11y/no-noninteractive-element-interactions
        <label className={inputLabelClass} onKeyDown={onKeyDown} tabIndex={0}>`
thoughts on which one you think we should use?

Reply:
my thinking is we want to be putting `role="button"` on the label

Reply:
gives the same type error with `role="button"`
![Screenshot 2024-08-01 at 12 37 31 PM](https://github.com/user-attachments/assets/668c88bd-08e7-4e89-a2eb-e6604e9d90e9)

Reply:
i think it narrowed it to a separate eslint error `jsx-a11y/no-noninteractive-element-to-interactive-role` which is more manageable

but we also disable the other two errors in a few of the elements already:
https://github.com/box/box-ui-elements/blob/219300fcff3e4cf594accbba5bab9575d633d55e/src/elements/content-preview/ContentPreview.js#L1317-L1318